Gemfile.lock

```diff
@@ -89,35 +98,45 @@ GEM
       money (~> 6.7)
       nokogiri (~> 1.5)
       sanitize (~> 2.1.0)
-    govuk-content-schema-test-helpers (1.6.0)
-      json-schema (~> 2.8.0)
-    govuk-lint (3.4.0)
+    govuk-content-schema-test-helpers (1.5.0)
```

Comment:
downgrading? is that deliberate?

Reply:
I've had a look into this and it looks like the `govuk_schemas` gem is the reason this is being downgraded, as we're using an early version of it which depends on an older version of `govuk-content-schema-test-helpers`.  I've had a quick look into what it would take to upgrade to a newer version of `govuk_schemas` and it looks a bit fiddly, so I'll create a separate card for this.